### ISSUE TITLE
Delayed Infinite Loop

### ISSUE THREAD

**Issue by astewes** [stevenwanderski/bxslider-3#222]
My horizontal slider spans 100% of the browser width - multiple (image) slides display across the viewport, with "moveSlides: 1" set in the options.

8 slides are being rotated. As I click the "next" arrow to progress through them, something odd is happening at the end. As the 8th slide initially enters the viewport and I continue clicking, there aren't any images displaying to the right of it UNTIL it's about cycle out again through the left side, at which point slides #1-3 suddenly appear where they should. From there, the cycle repeats, with the same scenario occurring once slide 8 becomes visible again. 

Basically, I want slide #1 and so forth to appear to the right of slide 8, without this lag. After inspecting the CSS, I'm pretty sure it's a matter of the viewport calculation being off (specifically, the "left" position of .bx-viewport). 

I know JS, but not to the level I'd need to to feel comfortable messing with the calculations in jquery.bxslider.js. Does anyone have any pointers?

**Comment by astewes** [comment]
I've replicated the issue in this fiddle: http://jsfiddle.net/astewes/S7L2n/1/

**Comment by patchgill** [comment]
Hi astewes,

Did you ever resolve this? I seem to be having the exact same issue. Any information would be greatly appreciated.
Cheers
J
